daffodil-test/src/test/resources/org/apache/daffodil/section23/dfdl_expressions/expressions.tdml

```diff
@@ -3252,14 +3252,9 @@
     model="newRegex" description="Section -  - DFDL-R">
 
     <tdml:document>45someletters</tdml:document>
-    <tdml:infoset>
-      <tdml:dfdlInfoset>
-        <s1>
-          <e1>45</e1>
-          <extra>someletters</extra>
-        </s1>
-      </tdml:dfdlInfoset>
-    </tdml:infoset>
+    <tdml:errors>
+      <tdml:error>This expression is not supported in the current option setting</tdml:error>
```

Comment:
This is kindof a weird error message, I"m not even sure what it means but it looks like the issue is the test is using a 4-byte unicode character in the regular expression? Is there extra information in the error message that the test just isn't looking for that makes the problem more clear? If not, it might be worth opening a bug to make this error message better, to at least make it clear something is wrong with the regular expression as opposed to  a DFDL expression.

Reply:
Not sure why I picked that as the error message to check for.  The error message does include more useful information and I have updated the test to check for that.